call_apply_bind_exercise/readme.md

```diff
@@ -8,14 +8,14 @@ var obj = {
     person: {
         sayHi: function(){
             return "This person's name is " + this.fullName
-        }
+        }.bind(obj)
```

Comment:
This won't work because obj won't be defined yet.  You can refactor this to:

```
var obj = {
  fullName: "Harry Potter",
  person: {
    sayHi( ) {
       return `This person's name is ${this.fullName}`;
    }
  }
};
obj.person.sayHi = obj.person.sayHi.bind(obj);